packages/react-composites/src/composites/localization/locales/en-US/strings.json

```diff
@@ -44,11 +44,12 @@
   },
   "callWithChat": {
     "peopleButtonLabel": "People",
-    "peopleButtonTooltipContentOpen": "Show Participants",
-    "peopleButtonTooltipContentClose": "Hide Participants",
+    "peopleButtonTooltipOpen": "Show participants",
+    "peopleButtonTooltipClose": "Hide participants",
     "chatButtonLabel": "Chat",
-    "chatButtonTooltipContentOpen": "Show Chat",
-    "chatButtonTooltipContentClose": "Hide Chat",
+    "chatButtonNewMessageCountToolTip": " ({unreadMessagesCount} unread)",
```

Comment:
I'd like to hear what others think, but I feel strings should be self contained and not appended to each other.
So this would be `chatButtonWithNewMessageCountTooltipOpen: "Show chat NewMessageCount"` -- but I'd liek to know what others think

Reply:
I agree with that. I actually thought that Donald was making that change, that's why haven't reviewed this PR today.

Reply:
ahhh okay, I will make that adjustment and keep that in mind for the future 😊